packages/ingester/src/ingest.ts

```diff
@@ -62,30 +67,58 @@ export async function ingestSources(
       for (const [key, potentialNewItem] of ingestibleItemsByItemIdentifiers) {
         if (!existingEntitiesByItemIdentifiers.get(key)) {
           // task is new
-          events.push(
+          ingestEvents.set(
+            key,
             createIngestTaskForSource(source, timeMillis)(potentialNewItem),
           );
+        } else {
+          // task exists
+          // TODO: we should make sure the content is up to date
```

Comment:
@andymatuschak I just realized that I haven't implemented this case yet. Does Orbit have an event for updating a `Task`'s content yet? Would you just issue a new `TaskIngestEvent` and associate it with the same entity id? Happy to dig into this further; just want to make sure this isn't a solved problem already.

Reply:
No, not yet! I was thinking something like a "spec update" event, probably with a sparse spec.